src/server/protocol.ts

```diff
@@ -174,6 +174,7 @@ export const enum CommandTypes {
     ProvideCallHierarchyOutgoingCalls = "provideCallHierarchyOutgoingCalls",
     ProvideInlayHints = "provideInlayHints",
     WatchChange = "watchChange",
+    MapCode = "mapCode",

```

Comment:
In general we have `-Full` requests for VS .. i am not sure if do that any more but something to think about.

Reply:
Well now even in VS we're not using the `-full` variations anymore because of our migration to LSP.

Reply:
This is a single request shared with VS Code. It's intended to be LSP-ish as-is.